webapi/src/main/scala/org/knora/webapi/store/triplestore/http/HttpTriplestoreConnector.scala

```diff
@@ -303,11 +314,22 @@ class HttpTriplestoreConnector extends Actor with ActorLogging with Instrumentat
       parseTry match {
         case Success(parsed) => Success(parsed)
         case Failure(e) =>
-          log.error(
-            e,
-            s"Couldn't parse response from triplestore:$logDelimiter$turtleStr${logDelimiter}in response to SPARQL query:$logDelimiter$sparql"
-          )
-          Failure(TriplestoreResponseException("Couldn't parse Turtle from triplestore", e, log))
+          if (turtleStr.contains("##  Query cancelled due to timeout during execution")) {
+            log.error(e, "Triplestore timed out while sending a response, after sending statuscode 200.")
+            Failure(
+              TriplestoreTimeoutException(
+                "Triplestore timed out while sending a response, after sending statuscode 200.",
+                e,
+                log
+              )
+            )
+          } else {
+            log.error(
+              e,
+              s"Couldn't parse response from triplestore:$logDelimiter$turtleStr${logDelimiter}in response to SPARQL query:$logDelimiter$sparql"
```

Comment:
Why does `logDelimiter` once is wrapped into `${...}` and once not? Can it be unified? I think both `$...` and `${...}` are the same.

Reply:
you're right, that's just how it is in the place where I copy-pasted it from... ;) I'll unify it

Reply:
ah, no... this can't be changed because right after the variable there is a word, so it would consider this part of the variable name, which then could not be found... I'll have to leave it as it is, even though it looks strange